latex_access/polish_braille.py

```diff
@@ -0,0 +1,391 @@
+        else:
+            translation =u"ć" +self.translate(arg[0]) +u"ę"
+        return (translation,arg[1])
+
+
+    def frac(self,input,start,rting=()):
+        '''Translates fractions into polish braille.
+
+        Returns touple as above'''
+        numerator=get_arg(input,start)
+        if numerator[1]<len(input):
+            denominator=get_arg(input,numerator[1])
+        else:
+            denominator=("",numerator[1],numerator[1])
+        if numerator[0].isdigit() and denominator[0].isdigit():
+            if input[start-6].isdigit() and start>5:
+                translation = "#"
+                if len(numerator[0]) <= 1:
+                    translation += self.upperNumbers[int(numerator[0])]
+                else:
+                    if numerator[0][0] != "-" and numerator[0][1].isdigit():
+                        for k in range(len(numerator[0])) : translation+=self.upperNumbers[int(numerator[0][k])]
+#                    translation = self.translate(translation,(rting[0]+numerator[2],rting[1]+1))
+            else:
+                translation=numerator[0]
+            for k in range(len(denominator[0])) : translation+=self.table["L"+denominator[0][k]]
+            if rting!=():
+                for j in range(len(numerator[0])): self.rt.append((rting[1]+j,rting[0]+numerator[2]+j))
+                for j in range(len(denominator[0])): self.rt.append((rting[1]+j+1+len(numerator[0]),rting[0]+denominator[2]+j))
+        else:
+            if rting==():
+                if numerator[0][0] == "-":
+                    self.lastnumber=-1
+                    translation=u"⠆"+"-"+self.translate(numerator[0].lstrip("-"))
+                    self.lastnumber=-1
+                    translation+=u"⠳"+self.translate(denominator[0])+u"⠰"
+                else:
+                    self.lastnumber=-1
+                    translation=u"⠆"+self.translate(numerator[0])
+                    self.lastnumber=-1
+                    translation+=u"⠳"+self.translate(denominator[0])+u"⠰"
+            else:
+                self.lastnumber=-1
+                transnum=self.translate(numerator[0],(rting[0]+numerator[2],rting[1]+1))
+                self.lastnumber=-1
+                transden=self.translate(denominator[0],(rting[0]+denominator[2],rting[1]+len(transnum)+2))
+                translation=u"⠆"+transnum+u"⠳"+transden+u"⠰"
+        return (translation,denominator[1])
+
+    def bar(self, input, start,rting=()):
+        '''Handles bar/overline.
+
+        Returns toutple'''
+        arg=get_arg(input,start)
+        if len(arg[0])==1:
+            translation=":%s" % arg[0]
+            if rting!=(): self.rt.append((rting[1]+1,rting[0]+arg[2]))
+        else:
+            if rting==(): translation=":{%so" % self.translate(arg[0])
+            else: translation=":{%so" % self.translate(arg[0],(rting[0]+arg[2],rting[1]+2))
+        return (translation,arg[1])
+
+    def tag(self,input,start):
+        '''Translate  tags into Nemeth.
```

Comment:
This still mentions Nemeth. Since I believe we don't know how tags should be translated in Polish Braille I think this method should be removed. Alternatively we may try to look up it in official notation reference, but I'm not sure if referencing equations is described there at all.